### DISCUSSION TITLE
List of strings - where is this type?

### DISCUSSION THREAD

**Discussion by OnkelTem** [mdelobelle/metadatamenu#577] [Q&A] [answered]
I cannot find a "list of strings" type.
What I see is:

![image](https://github.com/mdelobelle/metadatamenu/assets/114060/4a167a38-e792-4df7-af00-cbd8d0c0ae3a)

The first guess was Multi but it wants a list of **predefined** strings.

How to make a simple array of strings types? Like **authors of a book**.

**Comment by mdelobelle** [comment]
Yes this is "Multi" and you'll have to set the list of predefined strings.
You have several options for this: https://mdelobelle.github.io/metadatamenu/fields/#options_4

**Reply by OnkelTem** [reply]
But I don't want to pick anything from any list. It's just a field with names.
How to configure that one?

Examples:
- authors of the book
- aliases of a person

**Reply by mdelobelle** [reply, marked as answer]
There is no dedicated tupe for list of strings.
You could either:
- use an "input" type and format the list as you'd like
- use a "yaml" type if you want to put your list in the frontmatter of your note (yaml type field isn't allowed in the body of the note)
